pkg/config/schema.go

```diff
@@ -143,22 +143,27 @@ type ConfigAndStacksInfo struct {
 	AtmosBasePath             string
 }
 
+// Workflows
+
 type WorkflowStep struct {
+	Name    string `yaml:"name,omitempty" json:"name,omitempty" mapstructure:"name"`
```

Comment:
Can we  by default generate a friendly name, like `step1`, or `step2` based on the index in the list?